gammapy/data/observations.py

```diff
@@ -325,26 +325,29 @@ def ids(self):
         """List of obs IDs (`list`)"""
         return [str(obs.obs_id) for obs in self.list]
 
-    def select_time(self, time_interval):
+    def select_time(self, time_intervals):
         """Select a time interval of the observations.
 
         Parameters
         ----------
-        time_interval : `astropy.time.Time`
-            Start and stop time of the selected time interval.
-            For now we only support a single time interval.
+        time_intervals : list of `astropy.time.Time`
+            list of Start and stop time of the time intervals.
+
 
         Returns
         -------
         new_observations : `~gammapy.data.Observations`
-            A new observations instance of the specified time interval
+            A new observations instance of the specified time intervals
         """
         new_obs_list = []
+        if not isinstance(time_intervals, list):
```

Comment:
I think this can break if the user passes a list of two times?

rather do `isinstance(time_intervals,Time)`, this will force users to pass a `Time` object